lib/exercises.js

```diff
@@ -1,17 +1,49 @@
 // This method will return an array of arrays.
 // Each subarray will have strings which are anagrams of each other
-// Time Complexity: ?
-// Space Complexity: ?
+// Time Complexity: O(n * m)
+// Space Complexity: O(n)
 function grouped_anagrams(strings) {
```

Comment:
The Time complexity would be O(n) if the words aren't too long.  Or O(m * n log n) if the words are (m words and each word is at most n letters).  The sort is a mergesort and it costs the most with each word.